concreate/schema/overrides_schema.yaml

```diff
@@ -58,6 +58,13 @@ map:
           version: {type: str}
   dependencies:
     seq:
-      - map:
-          url: {type: str, required: True}
-          ref: {type: str}
\ No newline at end of file
+      - type: map
+        mapping:
+          name: {type: str, required: True}
+          git: 
+              type: map
+              mapping:
+                  url: {type: str, required: True}
+                  ref: {type: str}
+          directory: {type: str, required: False}
+        assert: "val['git'] is not None or val['file'] is not None"
```

Comment:
What is val['file'] here? Should it be val['directory'] ?

Reply:
Yes, but it doesn't look like assert is working since it's not generating errors. :/